app/src/main/res/layout/fragment_profile.xml

```diff
@@ -46,6 +46,13 @@
             android:drawableStart="@drawable/ic_email_black_24dp"
             android:hint="@string/msg_email"
             android:inputType="textEmailAddress" />
+        <EditText
+            android:id="@+id/text_avatar_url"
+            style="@style/EditText.Profile"
```

Comment:
Shouldn't this be `@style/Profile.EditText`, the one you defined doesn't exist!

Reply:
Yes, That was changed after me forking it and giving this PR..